client/quick-links/style.scss

```diff
@@ -0,0 +1,70 @@
+.woocommerce-page .woocommerce-quick-links {
+	$background-color: $white;
+
+	background-color: $background-color;
+
+	.components-card__body.is-size-large {
+		padding: 0;
+
+		.woocommerce-list {
+			margin: 0;
+			padding: 0;
+
+			.woocommerce-list__item {
+				// Ideally, we'd get these from `@wordpress/base-styles`.
+				// However, currently we are on an old version of `@wordpress/base-styles`,
+				// and updating that causes compatibility issues with the version of
+				// `@wordpress/components` that we are using, and updating that introduces
+				// breaking changes that we would need to address elsewhere in the
+				// `woocommerce-admin` codebase.
```

Comment:
Great documentation! Should we add to it by creating an issue and linking it here?

Reply:
I actually created an issue last week. I'll add a link to it in the code comment!